package.json

```diff
@@ -180,7 +180,7 @@
     "mnemonist": "^0.38.3",
     "pure-svg-code": "^1.0.6",
     "qs": "^6.10.1",
-    "radicle-contracts": "github:radicle-dev/radicle-contracts#commit=9fb23ac74a403b167a22513fc666779d871d292e",
+    "radicle-contracts": "github:radicle-dev/radicle-contracts#head=master",
```

Comment:
What are the advantages of pinning the dependency to `master`? IIUC this isn't a dependency which needs to be upgraded regularly with `yarn upgrade`, there are no security concerns. Any upgrades will be usually due to changes in the contracts' APIs, which may need adjustments in upstream. `yarn upgrade` may become harder to use if it will be risking fetching breaking changes from `radicle-contracts`. What we could do is create a git tag dependency, it'll be nicer to look at, but functionally equivalent to a hash.

Reply:
Excellent point. I changed it back to a commit. I don’t think there’s a need to create a tag. That would just create overhead.